app/tests/utils.py

```diff
@@ -11,3 +19,22 @@ def get_http_error_description(json_response):
     '''Get the HTTP error description from response
     '''
     return f"{json_response['description'] if 'description' in json_response else ''}"
+
+
+def get_sample_data(topic):
+    '''Get a dictionary of sample data as json
+
+    This function takes a string describing the subpath to the sample data and returns a
+    dictionary with key = filename, value = json.
+    f.ex. get_sample_data('collections')
+    :param topic: the name of the topic (f.ex. collections)
+    :return: dict
```

Comment:
I like this way of defining param and return values in docstrings however it doesn't match our guidelines. So I would make it match the guidelines but we might want also to rediscussed this guideline and change it ? @boecklic what is your opinion about this docstring guideline ?